Topic 0/src/point3/DemoHospital.java

```diff
@@ -0,0 +1,14 @@
+package point3;
+
+public class DemoHospital {
+
+	public static void main(String[] args) {
+		Factory hospitalfactory= new Factory();
+		Hospital hospital= hospitalfactory.getHospital("small");
```

Comment:
[Low] Instead of hardcoding *small* and *big* types, you can use Enum, and use the same Enumeration in your Factory. 
You can see an example here: http://crunchify.com/simple-java-enum-example/